### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by sh1l0n**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,10 +5,6 @@
 # Conventional directory for build outputs.
 build/
 
-# Omit committing pubspec.lock for library packages; see
```
Comment (tigloo) [inline .gitignore:8]: Shouldn't this stay omitted as per the guidelines quoted?

File: pubspec.yaml [hunk 2]
```diff
@@ -3,16 +3,17 @@ description: An internal tool for releasing dart apps
...
 
 dependencies:
-  args: 2.3.1
-  built_collection: 5.1.1
-  built_value: 8.4.0
-  pub_semver: 2.1.1
+  args: ^2.3.1
+  built_collection: ^5.1.1
+  built_value: ^8.4.4
+  meta: ^1.8.0
```
Comment (tigloo) [inline pubspec.yaml:12]: This was in dev_dependencies before, is this required in dependencies?

File: .gitignore (hunk 1 above)
Comment (sh1l0n) [reply .gitignore]: e03fd2c

File: pubspec.yaml (hunk 2 above)
Comment (vasbaz) [reply pubspec.yaml:12]: As I see, it was in **dependencies**. built_collection and built_value are not generator packages (like built_value_generator and build_runner, so it seems to me that they required here.

File: .gitignore (hunk 1 above)
Comment (vasbaz) [reply .gitignore]: I support the comment

**Review by vasbaz — APPROVED** [review]

**Pull request merged**